content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-flows.md

```diff
@@ -115,13 +117,13 @@ Use the `@CordaInject` annotation to define a field to be set by Corda before th
 
 In this sample CorDapp, add these services:
 
-* `FlowEngine`
-* `FlowIdentity`
-* `FlowMessaging`
-* `TransactionBuilderFactory`
-* `IdentityService`
-* `NotaryLookupService`
-* `JsonMarshallingService`
+* `FlowEngine`: Provides basic control over a flow it is injected into, such as calling subflows, performing asynchronous tasks and putting the flow to sleep for a period of time. This flow needs `FlowEngine` because you are utilizing subflows such as `FinalityFlow` and `CollectSignaturesFlow`.
+* `FlowIdentity`: Allows obtaining the identity of the node running the flow. You must inject this service because one of the output states needs its own identity.
```

Comment:
Do we need the "allows". Does it change the meaning if we simply say "Obtains" instead?
Same comment for points below.